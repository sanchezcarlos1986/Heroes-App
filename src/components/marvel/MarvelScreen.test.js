import React from 'react';
import {MarvelScreen} from '~components';
import {shallow} from 'enzyme';

describe('MarvelScreen', () => {
  let component;

  beforeEach(() => {
    component = shallow(<MarvelScreen />);
  });

  it('should renders ok', () => {
    expect(component).not.toBeNull();
    expect(component).toMatchSnapshot();
  });
});
