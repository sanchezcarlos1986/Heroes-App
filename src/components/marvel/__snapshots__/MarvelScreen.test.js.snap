// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarvelScreen should renders ok 1`] = `
<div>
  <h1>
    MarvelScreen
  </h1>
  <hr />
  <HeroList
    publisher="Marvel Comics"
  />
</div>
`;
